Plugins/SwiftLintPlugin/SwiftLintPlugin.swift

```diff
@@ -17,37 +17,30 @@
 //
 
 import Foundation
+
+#if canImport(PackagePlugin)
```

Comment:
Are you leaving package plugin supporting code in case Apple makes build plugins usable at some point?

Reply:
let‘s hope for this 🤞